src/Components/WriteArticle.jsx

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react'
 import { Form, Container, Button, Message, Dropdown } from 'semantic-ui-react'
 import { Redirect } from 'react-router-dom'
 import axios from 'axios'
+import { COUNTRY_OPTIONS } from './countriesData.js'
```

Comment:
Just out of interest, where are you taking these country options from?

Reply:
We installed an NPM package that actually creates a js file with all the countries and their 2 letter abbreviations. You can see the file inside the Components folder.

Reply:
> We installed an NPM package that actually creates a js file with all the countries and their 2 letter abbreviations. You can see the file inside the Components folder. The file is ready to be used as an options file of the `Dropdown` semantic component.

Nice! Good to know :)